### DISCUSSION TITLE
timescaledb error when moving database

### DISCUSSION THREAD

**Discussion by ched-dev** [directus/directus#12528] [Archive]
When I tried to `pg_dump` a database and import into another Postgres server, I got a lot of errors around `timescaledb` but it started with:

> ERROR: extension "timescaledb" does not exist

Is this a requirement to run Directus 9 or will my moved database be ok without it?

I can successfully connect and browse the new Postgres, so it seems fine without it?

**Comment by azrikahar** [comment]
Directus does not use/require `timemscaledb` extension, so it should be fine (at least in terms of Directus itself). On a semi-related note, the only extension that is needed to make maps/geometry fields work for Postgres is `postgis`.

Can't say I have a deep understanding here and more so an attempted guess, it should be when you did `pg_dump` earlier, the Postgres database does actually have `timescaledb` even if you weren't using it. You might be able to find a line such as `CREATE EXTENSION timescaledb;` in your dump file (once again I'm not sure if this is true). Then when you import it to another said Postgres server, which I can only assume is a different provider for example, it is trying to use the `timescaledb` extension but it in fact does not exist.

**Reply by ched-dev** [reply]
I exported the database from Railway and moved to another server. It's probably something enabled by default in Railway.

**Reply by azrikahar** [reply]
Thanks for pointing out it was a dump from Railway's PostgreSQL service! Seems like `timescaledb` (along with `postgis`) is indeed enabled by default in their offering: https://docs.railway.app/databases/postgresql#timescale-and-postgis
